components/phpunit_bridge.rst

```diff
@@ -0,0 +1,129 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report trigerred errors,
+    legacy tests and usage of deprecated code.
```

Comment:
Do we want to talk about it still? The bootstrap files allow to set the locale and the annotations (which is mentioned just below). As we do mention that it automatically registers an event listener, do we need to add that it's called `SymfonyTestListener`?

For the `ClockMock`, I couldn't find any doc on it. What is it supposed to do in the bridge?